08-transport.Rmd

```diff
@@ -203,21 +218,24 @@ The result is an updated `zones` dataset that contains new columns representing
 The geographic distribution of trip origins is illustrated in the left-hand map in Figure \@ref(fig:zones).
 This shows that most zones have between 0 and 4,000 trips originating from them in the study area.
 More trips are made by people living near the center of Bristol and fewer on the outskirts.
-Why is this? Remember that we are only dealing trips within the study region:
-low trip numbers in the outskirts of the region can be explained by the fact that many people in these peripheral zones will travel to other regions.
+<!-- What is the reasoning for this? I mean since it is an aggregate we could also consider all trips made regardless of the fact if the destiation is within the study area. -->
+Why is this? Remember that we are only dealing with trips within the study region:
```

Comment:
What is the reasoning for this? I mean since it is an aggregate we could also consider all trips made regardless of the fact if the destiation is within the study area.